charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 17.1.3
+version: 17.2.1
```

Comment:
@invidian This should be: `17.2.0`.

Reply:
I forgot the dummyness of charts release process :) Done dashboards syncing before adding a feature, without version bumping then.